util/ranger/ranger.go

```diff
@@ -466,3 +467,33 @@ func newFieldType(tp *types.FieldType) *types.FieldType {
 		return tp
 	}
 }
+
+func points2EqOrInCond(ctx sessionctx.Context, points []point, expr expression.Expression) expression.Expression {
```

Comment:
we need a comment for this function to explain:
1. the basic functionality. 
2. the constraints of the input parameters.

how about:
```go
// points2EqOrInCond constructs a 'EQUAL' or 'IN' scalar function based on the
// 'points'. The target column is extracted from the 'expr'.
// NOTE:
// 1. 'expr' must be either 'EQUAL' or 'IN' function.
// 2. 'points' should not be empty.
```